src/formatter_md.py

```diff
@@ -113,3 +113,39 @@ def add_horizontal(self):
                 horizontal(str): horizontal rule string
         """
         return '---'
+
+    def _prepare_headers_cells(self, headers):
+        """Prepare headers cells for table creation.
+
+        Args:
+            headers (list): table column names
+
+        Returns:
+            headers_cells(list): headers cells ready for further table-related
+            operations
+        """
+        headers_row = ['{header} | '.format(header=h) for h in headers]
+        headers_row = ''.join(headers_row)
+        headers_separation = '--- | ' * len(headers)
+        headers_cells = [headers_row, headers_separation]
+        return headers_cells
+
+    def create_table(self, rows, headers):
+        """ Creates table with the given size.
+
+        Args:
+           rows(list): list of lists; each nested list is a table row
+           headers(list): column names
+
+        Returns:
+            table(list): list of formatted strings; every string represents
+            single row, starting from headers
+        """
+        headers_cells = self._prepare_headers_cells(headers)
```

Comment:
can be:
table = list()
table += [' | '.join(headers), '--- | ' * len(headers)]
for row in rows:
    table.append(' | '.join(row))
return table

Reply:
done